applications/GeoMechanicsApplication/tests/test_constant_strip_load_2d/ProjectParameters.json

```diff
@@ -0,0 +1,194 @@
+{
+    "problem_data": {
+        "problem_name": "test_constant_strip_load_2d",
+        "start_time": 0.0,
+        "end_time": 1,
+        "echo_level": 1,
+        "parallel_type": "OpenMP",
+        "number_of_threads": 4
+    },
+    "solver_settings": {
+        "solver_type": "U_Pw",
+        "model_part_name": "PorousDomain",
+        "domain_size": 2,
+        "model_import_settings": {
+            "input_type": "mdpa",
+            "input_filename": "test_constant_strip_load_2d"
+        },
+        "material_import_settings": {
+            "materials_filename": "MaterialParameters_stage_1.json"
+        },
+        "time_stepping": {
+            "time_step": 0.01,
+            "max_delta_time_factor": 1000
+        },
+        "reduction_factor": 1.0,
+        "increase_factor": 1.0,
+        "buffer_size": 2,
+        "echo_level": 1,
+        "clear_storage": false,
+        "compute_reactions": false,
+        "move_mesh_flag": false,
+        "reform_dofs_at_each_step": false,
+        "nodal_smoothing": true,
+        "block_builder": true,
+        "rebuild_level": 0,
+        "prebuild_dynamics": true,
+        "solution_type": "dynamic",
+        "rayleigh_m": 0.0001,
+        "rayleigh_k": 0.0001,
+        "calculate_reactions": true,
+        "rotation_dofs": true,
+        "reset_displacements": false,
+        "scheme_type": "newmark",
+        "newmark_beta": 0.25,
+        "newmark_gamma": 0.5,
+        "newmark_theta": 0.5,
+        "strategy_type": "newton_raphson",
+        "max_iterations": 15,
+        "min_iterations": 6,
+        "number_cycles": 1,
+        "convergence_criterion": "displacement_criterion",
+        "displacement_relative_tolerance": 1e-09,
+        "displacement_absolute_tolerance": 1e-12,
+        "linear_solver_settings": {
+            "solver_type": "amgcl",
+            "scaling": false,
+            "tolerance": 1e-06,
+            "max_iteration": 1000
+        },
+        "problem_domain_sub_model_part_list": [
+            "soil_block"
+        ],
+        "processes_sub_model_part_list": [
+            "load",
+            "base_fixed",
+            "side_rollers",
+            "zero_water_pressure"
+        ],
+        "body_domain_sub_model_part_list": [
+            "soil_block"
+        ]
+    },
+    "output_processes": {
+    },
+    "processes": {
+        "json_output": [
+            {
+                "python_module": "json_output_process",
+                "kratos_module": "KratosMultiphysics",
+                "process_name": "JsonOutputProcess",
+                "Parameters": {
+                    "model_part_name": "PorousDomain.json_output",
+                    "output_file_name": "json_output.json",
+                    "output_variables": [
+                    ],
+                    "gauss_points_output_variables": [
+                        "CAUCHY_STRESS_VECTOR"
+                    ],
+                    "time_frequency": 0.9999999999
```

Comment:
I dont think it can be 1.0, as Kratos only writes the output to the json_file right after  this "time frequency". If I fill in 1.0, no output will be written.

The output which is written currently, is at time 1.0